src/js/components/AppPageComponent.jsx

```diff
@@ -157,7 +157,16 @@ var AppPageComponent = React.createClass({
     });
   },
 
-  onScaleAppError: function (errorMessage) {
+  onScaleAppError: function (errorMessage, statusCode, instances) {
+    if (statusCode === 409) {
+      var appId = this.state.appId;
+      const dialogId = DialogActions.
+        confirm("App scaling failed.Do you want to try forcing?");
```

Comment:
Let's add a little bit to this:
`"Failed to scale ${appId}. Do you want to try forcing the scale action?"`

I think it might make sense to also add a little bit of explanation here. Something like:
"`Forcing the scale action will stop any current deployment of the app, and start a new one to scale it"`

With this it does make sense to have the `appId` variable, but let us use `let appId` to declare it.